.circleci/continue_config.yml

```diff
@@ -187,16 +222,19 @@ workflows:
   ci:
     jobs:
       - ci-base:
-          name: ci-base-<< matrix.python-version >>
+          name: ci-base-<< matrix.python-version >><< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
+              extras: ["", "-metrics"]
       - unit-test:
+          name: unit-test-<< matrix.python-version >><< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
+              extras: ["", "-metrics"]
           requires:
-            - ci-base-<< matrix.python-version >>
+            - ci-base-<< matrix.python-version >><< matrix.extras >>
```

Comment:
For consistency, please also update the `requires` and/or `matrix.extras` to integration test and other jobs that also depends on `ci-base`.